src/main/java/net/sf/jabref/gui/fieldeditors/JTextAreaWithHighlighting.java

```diff
@@ -77,39 +77,37 @@ public void undoableEditHappened(UndoableEditEvent evt) {
         });
 
         // Create an undo action and add it to the text component
-        getActionMap().put("Undo",
-                new AbstractAction("Undo") {
-
-                    @Override
-                    public void actionPerformed(ActionEvent evt) {
-                        try {
-                            if (undo.canUndo()) {
-                                undo.undo();
-                            }
-                        } catch (CannotUndoException ignored) {
-                    // Ignored
-                        }
+        getActionMap().put("Undo", new AbstractAction(Actions.UNDO) {
```

Comment:
Actions.UNDO == "undo" whereas you had "Undo" before. I am not sure which effects will this change have. We should be careful here. Does it still work for this special component?